site/content/en/docs/faq.md

```diff
@@ -0,0 +1,275 @@
+---
+title: "FAQ"
+weight: 120
+description: >
+  Frequently asked questions and answers
+---
+
+# FAQ Section
+
+Frequently asked questions from newcomers to the Prow repository.
+
+## General Questions
+
+### What is Prow?
+
+Prow is a Kubernetes-based Continuous Integration and Continuous Deployment (CI/CD) system. It provides automated testing, code review automation, and project management features for Kubernetes and other open-source projects.
+
+### Who maintains Prow?
+
+The Kubernetes SIG Testing team maintains Prow. See the [OWNERS](https://github.com/kubernetes-sigs/prow/blob/main/OWNERS) file for the list of maintainers.
+
+### What programming language is Prow written in?
+
+Primarily Go (94.7%), with some TypeScript (2.5%) for the Deck frontend.
```

Comment:
I do not think we need percentages of languages included in docs. Especially since GitHub gives this data automatically.